packages/jest-message-util/src/index.ts

```diff
@@ -283,7 +284,7 @@ export const formatStackTrace = (
     ? slash(path.relative(config.rootDir, testPath))
     : null;
 
-  if (topFrame) {
+  if (topFrame && !options.noCodeFrame) {
```

Comment:
do we need the `getTopFrame` call if `noCodeFrame: true` is passed, or can we drop it?

Reply:
We do not need to call `getTopFrame` if `noCodeFrame: true` is passed. Added an `if ` condition outside. Wonder if there are ways to reduce the number of nested `if`s here 🤔